tools/star.wdl

```diff
@@ -120,13 +122,26 @@ task build_star_db {
 task alignment {
     meta {
         description: "This WDL task runs the STAR aligner on a set of RNA-Seq FASTQ files."
+        external_help: "https://github.com/alexdobin/STAR/blob/2.7.10a/doc/STARmanual.pdf"  # TODO keep this up to date with container updates
+        outputs: {
+            star_log: "Summary mapping statistics after mapping job is complete. The statistics are calculated for each read (single- or paired-end) and then summed or averaged over all reads. Note that STAR counts a paired-end read as one read. Most of the information is collected about the UNIQUE mappers. Each splicing is counted in the numbers of splices, which would correspond to summing the counts in SJ.out.tab. The mismatch/indel error rates are calculated on a per base basis, i.e. as total number of mismatches/indels in all unique mappers divided by the total number of mapped bases."
+            star_bam: "STAR aligned BAM"
+            star_junctions: {
+                description:"File contains high confidence collapsed splice junctions in tab-delimited format. Note that STAR defines the junction start/end as intronic bases, while many other software define them as exonic bases. See `external_help` for file specification."
+                external_help: "https://github.com/alexdobin/STAR/blob/2.7.10a/doc/STARmanual.pdf"  # keep this up to date with container updates
```

Comment:
Is this going to be difficult to maintain? I wonder if for STAR `external_help` should just be specified once at the top level, rather than per-output.

Reply:
I think that's probably a good idea. If we do that, I think it would be appropriate to change the last sentences here to something along the lines of "See `meta.external_help` for file specification."
Question though, is `meta.external_help` the correct way to refer to that section (from here)? I can't think of a good alternative.

Reply:
I think that's about the best I can come up with. If WDLdoc can handle that specially and just link to it, that would make it easier when looking at the rendered docs.

Reply:
That would be a neat feature. Will try and remember that for when I get to the WDLdoc update.